features/issues/components/issue-list/issue-list.tsx

```diff
@@ -73,25 +129,46 @@ const PageNumber = styled.span`
 
 export function IssueList() {
   const router = useRouter();
+
+  const [status, setOptionStatus] = useState("");
+  const [level, setOptionLevel] = useState("");
```

Comment:
You don't really need these states. You can just set the value directly to the URL

Reply:
Sorry, I don't understand... when I delete those useStates, the type checks on the useIssues arguments, are throwing errors. 🗡️ 

And I wouldn't be able to use the setter functions for the onChange event on the select...

![level](https://user-images.githubusercontent.com/108257596/215309983-ee470a15-c50e-4d6c-a0c1-efbed4783173.JPG)

Reply:
Haha! I don't mean you can delete the state and it will work out of the box. The variable `status` and `level` need to come from somewhere, right? And you already have them in the URL, don't you?